data/ElizabethOgbee.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Elizabeth Ogbe",
+  "type": "personal",
+  "bio": "Blockchain Deverloper",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/elizabeth_ogbe",
+      "icon": "FaTwitter"
+    }
+  ]
+  "links": [
+    {
+      "name": "Github",
+      "url": "https://github.com/ElizabethOgbee",
+      "icon": "FaGithub"
+    }
+  ]
```

```suggestion
  "links": [
    {
      "name": "Follow me on Twitter",
      "url": "https://twitter.com/elizabeth_ogbe",
      "icon": "FaTwitter"
    },
    {
      "name": "Github",
      "url": "https://github.com/ElizabethOgbee",
      "icon": "FaGithub"
    }
  ]
```


I noticed that you have used the "links" key twice in your JSON object. 

To avoid any confusion or errors, it's recommended to use the "links" key only once and include all the links in that array.